CMakeLists.txt

```diff
@@ -278,6 +284,14 @@ find_program(MIOPEN_AMDGCN_ASSEMBLER
 )
 message(STATUS "AMDGCN assembler: ${MIOPEN_AMDGCN_ASSEMBLER}")
 
+# miopentensile
+if(MIOPEN_USE_MIOPENTENSILE)
+    find_package(miopentensile)
+    message(STATUS "Build with miopentensile")
+else()
+    message(STATUS "Build without miopentensile")
```

Comment:
I think that cmake should issue an error message and fail here. /cc  @pfultz2

Reply:
Not really.
The failure was causing by dependency installation.

Reply:
I mean that if the user wants to use miopentensile (`-DMIOPEN_USE_MIOPENTENSILE=On`) and package not found, then cmake should fail.

Reply:
Do you mean cmake should fail with error msg if MIOPEN_USE_MIOPENTENSILE is ON but package is not found.

Reply:
Considering that the tensile eco is still under construction, I suggest not issuing error msg but just msg to warn user that it is not built with miopentensile. When miopentensile becomes the only gemm backend, we will issue the error msg. @atamazov @pfultz2

Reply:
I still think that when cmake is explicitly asked to do something but can't do that it should fail. I found this useful in real life. Please look at [similar case at initial COMGR PR](https://github.com/ROCmSoftwarePlatform/MIOpen/pull/177#discussion_r416088277).

@pfultz2 What is your opinion?

Reply:
Okay, let's keep it for now. However please keep in mind that in case of missing miopentensile in out CI environment, the tensile tests would pass, but actual testing wouldn't happen.